models/user_model.py

```diff
@@ -93,6 +94,10 @@ def delete_user(username):
         db.session.commit()
         return done
 
+    def update_last_login(self):
```

Comment:
New method update_last_login() requires unit tests. Add tests for timestamp updates and database commit.